org.musicbrainz.Picard.appdata.xml.in

```diff
@@ -20,10 +20,47 @@
     <p>
       Tagging audio files has never been easier.
     </p>
+    <p>Features:</p>
+    <ul>
+      <li>
+        Multiple formats: Picard supports all popular music formats,
+        including MP3, FLAC, OGG, M4A, WMA, WAV, and more.
+      </li>
+      <li>
+        AcoustID: Picard uses AcoustID audio fingerprints, allowing files
+        to be identified by the actual music, even if they have no metadata.
+      </li>
+      <li>
+        Comprehensive database: Picard uses the open and
+        community-maintained MusicBrainz database to provide accurate
+        information about over 2 million music releases.
```

Comment:
I wouldn't use "2" here, even if correct, since it will increase over the time, and we don't want to update this string every time. I'd rather use a generic "over millions of music releases".